examples/samples/messageBrowse.ts

```diff
@@ -0,0 +1,125 @@
+import {
+  OnMessage,
+  OnError,
+  MessagingError,
+  delay,
+  ServiceBusMessage,
+  ReceiveMode,
+  generateUuid,
+  Namespace
+} from "../../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const path = process.env.QUEUE_NAME || "";
+
+console.log("str: ", str);
+console.log("path: ", path);
+
+async function main(): Promise<void> {
+  // send a set of messages
+  await sendMessage();
+  // browse those messages
+  await peekMessage();
+  // receive the messages [ to clear the queue ]
+  await receiveMessage();
+}
```

Comment:
We are dealing with multiple messages in this sample, but the names of the send/peek/receive suggest that we are dealing with a single message